scripts/dream.py

```diff
@@ -255,8 +314,27 @@ def image_writer(image, seed, upscaled=False):
                         results.append([path, formatted_dream_prompt])
                 last_results.append([path, seed])
 
+                if opt.save_progress is not None:
+                    nonlocal progress_video_type
+                    if progress_video_type == 'v' or progress_video_type == 'vo':
+                            frames_for_video.append(image)
+                            make_video(frames_for_video, os.path.join(current_outdir, f'{prefix}.{seed}.mp4'))
+                            frames_for_video.clear()
+
+                if opt.show_progress is not None:
+                    show_progress(image)
+                    import cv2
+                    cv2.waitKey(int(opt.show_progress[1]) * 1000)
+                    cv2.destroyAllWindows()
```

Comment:
I would suggest you remove this code and the partial option for it. The experience I get with it removed is good. `-show_progress 3` opens the window and updates as it goes. At the end the window remains. If I prompt again with `-show_progress` same window is re-used. Finally, when I'm done I can just close the window manually.
If I decide to abort midway then the window stays open. I can try another prompt, works.

Reply:
I could do that but it becomes unresponsive when it is no longer being used. Which I'm not sure if there's a way to bypass using cv2.

Reply:
I tested a bit more and now I also saw the unresponsiveness. I even tried a couple of quick fixes like moving cv2.waitKey to a separate thread which didn't work. I think a separate process might work. Regardless, deferring this to you and other reviewers.

Reply:
I've tried threading but cv2 refuses to comply. I've looked it up and seemingly there's no obvious solution to the unresponsiveness when the waitKey is not handled.